rest-service/manager_rest/dsl_functions.py

```diff
@@ -32,86 +32,113 @@
 SecretType = namedtuple('Secret', 'key value')
 
 
+def evaluate_node(node):
+    node['name'] = node['id']
```

Comment:
why do we need this? did you switch to use id?

Reply:
yes, dsl-parser uses 'name', while the storage uses 'id'. Added a comment saying that